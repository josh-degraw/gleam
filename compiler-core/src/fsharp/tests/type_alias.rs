use crate::assert_fsharp;

#[test]
fn type_alias() {
    assert_fsharp!(
        r#"
pub type Headers = List(#(String, String))
"#,
    );
}
