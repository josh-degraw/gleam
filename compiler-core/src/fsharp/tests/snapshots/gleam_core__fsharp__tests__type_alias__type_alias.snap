---
source: compiler-core/src/fsharp/tests/type_alias.rs
expression: "\npub type Headers = List(#(String, String))\n"
---
module rec my.``mod``
type Headers = list<(string * string)>
